context/on_startup.py

```diff
@@ -30,15 +30,20 @@ def write_igv_configuration():
         assembly
     )
     for node_data in config_data["node_info"].values():
-        tracks.append(
-            {
-                "name": "{} - {}".format(
-                    node_data["node_solr_info"]["name"],
-                    node_data["file_url"]
-                ),
-                "url": node_data["file_url"]
-            }
-        )
+        track = {
+            "name": "{} - {}".format(
+                node_data["node_solr_info"]["name"],
+                node_data["file_url"]
+            ),
+            "url": node_data["file_url"]
+        }
+        if '.bam' == os.path.splitext(track['name']):
```

Comment:
```suggestion
        if '.bam' == os.path.splitext(node_data["file_url"]):
```
Probably safer to go to the source for this: The `track['name']` is really just a human readable string, and the format could change.

(If you do accept this, maybe define a variable for `node_data["file_url"]`, since it's being used three times.)

Reply:
done!